lua/defaults/override.lua

```diff
@@ -0,0 +1,48 @@
+-- Map leader key e.i. <space>
+vim.g.mapleader = ' '
+vim.g.maplocalleader = ' '
+
+-- Enable/Disable text highlighting on search
+vim.o.hlsearch = true
+
+-- Enable/Disable line numbers
+vim.wo.number = true
+
+-- Enable/Disable line wrap
+vim.wo.wrap = false
+
+-- Enable/Disable Mouse
+vim.o.mouse = 'a'
+
+-- Enable/Disable unified system clipboard
+vim.o.clipboard = 'unnamedplus'
+
+-- Enable/Disable break indent
+vim.o.breakindent = true
+
+-- Enable/Disable keeping an undo history of each file
+vim.o.undofile = true
+
+-- Enable/Disable Case-insensitive searching
+-- NOTE: \C or capital in search will override this setting
+vim.o.ignorecase = true
+vim.o.smartcase = true
+
+-- Enable/Disable Single Column Mode
+vim.wo.signcolumn = 'yes'
```

Comment:
this is wrong it doesn't have anything to do with single column mode.
this enable the side gutter on the buffer with signs can be displayed.
see :help signcolumn

Reply:
resolved 0c03153
It should be handled by other packages/plugins that use the column